infertrade/algos/community/allocations.py

```diff
@@ -1091,7 +1137,7 @@ def create_infertrade_export_allocations():
         infertrade_export_allocations_raw.update(
             {
                 function.__name__: {
-                    "function": function,
+                    "function": str(function),
```

Comment:
@Optimus-Q - why is this change needed for pure rule additions? This is a bug fix? 

@SaurabhBansalInferstat - will this impact any active downstream usages?

Reply:
Changed back to old.